src/main/java/com/internet/shop/controllers/GetAllProductsAdminController.java

```diff
@@ -0,0 +1,27 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.ProductService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetAllProductsAdminController extends HttpServlet {
```

Comment:
same

Reply:
I have GetAllProductsController (whre user can see all products amd buy it) already.
so i cant rename it.